### PR TITLE
DONE! wewewewewew

### PR DESCRIPTION
**Pull request by liznomura**

### CODE REVIEW

File: secret_number.js [hunk 1]
```diff
@@ -1,4 +1,9 @@
 'use strict';
 module.exports = function() {
-  
+  var secretNumber = 0;
+  var randomNumber = Math.random() * (1000000);
```
Comment (jesse-copeland) [inline secret_number.js]: This solution will never get 1000000

File: secret_number.js [hunk 2]
```diff
@@ -1,4 +1,9 @@
...
+  return function() {
+    secretNumber = randomNumber;
```
Comment (jesse-copeland) [inline secret_number.js]: this assignment does not add much value

**Review by jesse-copeland — CHANGES_REQUESTED** [review]
see notes

**Comment by taesup** [comment]
Looks good! Feel free to merge this into master.

**Pull request merged**